src/main/java/com/eclipsesource/v8/V8Array.java

```diff
@@ -610,6 +610,60 @@ public V8Array pushUndefined() {
         return this;
     }
 
+    /**
+     * Pushes all integers in the given array to the next available spots in the Array.
+     *
+     * @param values The integer array to push to the array.
+     * @return The receiver.
+     */
+    public V8Array pushIntegers(final int[] values) {
+        v8.checkThread();
+        checkReleased();
+        if (values == null) {
+            throw new NullPointerException("values is null");
+        }
+        for (int value : values) {
+            push(value);
```

Comment:
This for-loop is very slow because there are N JNI calls. V8 supports `push(element1, element2, /* …, */ elementN)` so that there is only 1 JNI call.